packages/css/src/types.ts

```diff
@@ -120,7 +120,11 @@ export interface Composition {
   classList: string;
 }
 
-type IdentOption = 'short' | 'debug';
+type IdentOption =
+  | 'short'
+  | 'debug'
+  | ((scope: string, refCount: string, debugId: string | undefined) => string);
```

Comment:
Few thoughts about this:
- I think this should instead be an object for named params. 
- I think `scope` is too abstract, seeing as this is a hash of the file path. I think it'd be more useful to provide a general hash for the identifier which would be `scope + refCount`. Let's just call it `hash`.
- Maybe also add filename and packageName to be fed into a custom hash

Reply:
Thank for the suggestion, I updated its type and added the extra params suggested, also for the filename I think it makes sense to just pass the file path and give the freedom to the developer to parse this file path or X to retrieve himself the filename or do X operations on it.